docs/servo_power_test/src/total_histogram.ts

```diff
@@ -0,0 +1,147 @@
+import * as d3 from 'd3';
+
+export class TotalHistogram {
+  private colorPalette = [
+    'rgba(66, 133, 244, 1)', // #4285f4
```

Comment:
If these alpha values are always 1.0, they should be a color codes directly.
c.f.
- https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation
- https://google.github.io/styleguide/htmlcssguide.html#Capitalization